scripts/homestead.rb

```diff
@@ -9,6 +9,15 @@ def self.configure(config, settings)
 
     # Allow SSH Agent Forward from The Box
     config.ssh.forward_agent = true
+      
```

Comment:
can we simplify this to use our default in the `else` conditional? also, we need to use input from the settings.  not sure about the naming though....

```suggestion
    # Configure Verify Host Key
    if settings.has_key?('verify_host_key')
      config.ssh.verify_host_key = settings['verify_host_key']
    else
      config.ssh.verify_host_key = true
    end
```